types/src/lib.rs

```diff
@@ -0,0 +1,57 @@
+pub use crate::data_types::{Column, DataTypeNode};
+use crate::decoders::{ensure_size, read_string};
+use crate::error::TypesError;
+pub use crate::leb128::put_leb128;
+pub use crate::leb128::read_leb128;
```

Comment:
`read_leb128` but `parse_rbwnat_...`. Is there some special meaning here?

Reply:
Reading LEB128 also means decoding. I'm just pointing out the inconsistency.
We can also use "serialize_rbwnat_header"/"deserialize_rbwnat_header".

Anyway, you can resolve the comment in any state. This is an additional crate, not supposed to be used directly. And, I still not sure why serializer/deserializer for data and headers are stored in the different modules, but it's ok for me while it's not the public API of the main crate.